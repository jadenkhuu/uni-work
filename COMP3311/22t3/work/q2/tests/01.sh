psql racing -c 'select * from q2 order by name;'
