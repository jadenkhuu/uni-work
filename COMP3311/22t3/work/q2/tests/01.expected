         name         |   course   |    date    
----------------------+------------+------------
 Blue Sapphire Stakes | Caulfield  | 2019-10-16
 Caulfield Cup        | Caulfield  | 2019-10-19
 Race 2               | Rosehill   | 2019-11-09
 Race 2               | Rosehill   | 2020-03-21
 Race 3               | Flemington | 2019-11-07
 Race 3               | Doomben    | 2020-06-13
(6 rows)
